src/state_transition/cache/epoch_cache.zig

```diff
@@ -450,6 +444,30 @@ pub const EpochCache = struct {
         self.sync_period = computeSyncPeriodAtEpoch(self.epoch);
     }
 
+    /// At fork boundary, this runs post-fork logic and after `upgradeState*`.
+    pub fn finalProcessEpoch(self: *EpochCache, cached_state: *const CachedBeaconStateAllForks) !void {
+        const state = cached_state.state;
+
+        // Post-Fulu, EIP-7917 introduced the `proposer_lookahead`
```

Comment:
miss assigning `proposers_prev_epoch` to `proposers`

Reply:
Fixed in [6693e3e](https://github.com/ChainSafe/lodestar-z/pull/142/commits/6693e3eb11e2748524c8f169fd704b6c8ec86b97)